AggregateMetrics/AggregateMetrics/AzureWebApp/PerformanceCollectorModule.cs

```diff
@@ -21,14 +24,40 @@ public class PerformanceCollectorModule : ITelemetryModule
                                                                 @"\Processor(_Total)\% Processor Time"
                                                             };
 
-        public IList<PerformanceCounterCollectionRequest> Counters { get; private set; }
+        public List<PerformanceCounterCollectionRequest> Counters { get; private set; }
```

Comment:
@mateoatr the error could be because there is no default initialization of the list variable in a constructor.  Making collections as properties directly settable is generally a bad design.